### PR TITLE
MAINTAINER file terminology change: remove orphaned usage

### PR DESCRIPTION
**Pull request by nashif**
- ci: compliance: skip codeowner checks
- boards: add maintainer to the platform yaml file
- MAINTAINERS: do not use orphaned terminology

### CODE REVIEW

File: MAINTAINERS.yml [hunk 1]
```diff
@@ -320,9 +319,9 @@ CMSIS-DSP integration:
         - "area: CMSIS-DSP"
 
 Common arch:
-    status: orphaned
-    collaborators:
+    maintainers:
         - dcpleung
+    collaborators:
         - nashif
```
Comment (stephanosio) [inline MAINTAINERS.yml:325]: Should this be a separate commit/reviewed by the TSC since we are adding a maintainer here?

Also you removed `status:` here completely, which is not consistent with the rest (other maintained areas have `status: maintained`).

File: MAINTAINERS.yml [hunk 2]
```diff
@@ -698,7 +697,6 @@ Documentation:
         - "area: IEEE 802.15.4"
 
 "Drivers: Interrupt controllers":
-    status: orphaned
     files:
         - drivers/interrupt_controller/
```
Comment (stephanosio) [inline MAINTAINERS.yml:701]: Did you mean to add `status: odd fixes` here?

File: MAINTAINERS.yml [hunk 3]
```diff
@@ -782,7 +780,6 @@ Documentation:
         - "area: Modem"
 
 "Drivers: Neural Networks":
-    status: orphaned
     collaborators:
         - nashif
     files:
```
Comment (stephanosio) [inline MAINTAINERS.yml:785]: same here

File: MAINTAINERS.yml [hunk 4]
```diff
@@ -1414,8 +1411,7 @@ Shields:
         - "area: Shields"
 
 SPARC arch:
-    status: orphaned
-    collaborators:
+    maintainers:
         - martin-aberg
```
Comment (stephanosio) [inline MAINTAINERS.yml:1415]: also adding maintainer here (separate commit/TSC review?)

also removing `status:` here completely.

File: MAINTAINERS.yml (hunk 1 above)
Comment (nashif) [reply MAINTAINERS.yml]: yep, remove those for now, will come in a different PR

**Review by henrikbrixandersen — APPROVED** [review]

File: MAINTAINERS.yml [hunk 5]
```diff
@@ -13,9 +13,8 @@
 #             The area has a Maintainer (approved by the TSC) who
 #             looks after the area.
 #
-#         * orphaned:
-#             No current maintainer (but maybe you could take the role as you
-#             write your new code).
+#         * odd fixes:
+#            The ares gets odd fixes and has collaborators.
```
Comment (JordanYates) [inline MAINTAINERS.yml:17]: ares/area

**Review by MaureenHelm — APPROVED** [review]

**Review by keith-zephyr — APPROVED** [review]
